DailyCodingProblem/Models/Node.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DailyCodingProblem.Models
+{
+    public class Node
+    {
+        public int Value { get; set; }
+
+        public Node LeftChild { get; set; }
+
+        public Node RightChild { get; set; }
+
+        public Node(int value, Node leftChild, Node rightChild)
```

Comment:
Since you don’t need the other 2 parameters half the time, it would be nice to make a second constructor with only one parameter